Tests/SotoCoreTests/AWSClientTests.swift

```diff
@@ -401,14 +399,14 @@ class AWSClientTests: XCTestCase {
             let threadPool = NIOThreadPool(numberOfThreads: 3)
             threadPool.start()
             let fileIO = NonBlockingFileIO(threadPool: threadPool)
-            let fileHandle = try fileIO.openFile(path: filename, mode: .read, eventLoop: httpClient.eventLoopGroup.next()).wait()
+            let fileHandle = try await fileIO.openFile(path: filename, mode: .read, eventLoop: httpClient.eventLoopGroup.next()).get()
```

Comment:
We could really do with a new async version of FileIO, preferably with an AsyncSequence